contracts/nois-payment/src/msg.rs

```diff
@@ -0,0 +1,37 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::Uint128;
+
+use crate::state::Config;
+
+#[cw_serde]
+pub struct InstantiateMsg {
+    pub nois_sink: String,
+    pub nois_com_pool_addr: String,
+}
+
+#[cw_serde]
+pub enum ExecuteMsg {
+    /// Pays by distributing the funds according to what has been instructed by the gateway
+    Pay {
+        burn: Uint128,
+        community_pool: Uint128,
+        relayer: (String, Uint128),
```

Comment:
Let's use `Coin` type for all 3 values. Then this contract can easily support other tokens as well. Also we don't need to hardcode the denom anymore but gateway controls all pricing.

```suggestion
        burn: Coin,
        community_pool: Coin,
        relayer: (String, Coin),
```